pkg/management/component.go

```diff
@@ -1573,6 +1573,12 @@ func (c *component) GetRole(ctx context.Context, realmName string, roleID string
 	roleRep.ContainerID = roleKc.ContainerID
 	roleRep.Description = roleKc.Description
 
+	// Filter out the business role flag
+	attributes := roleKc.Attributes
+	delete(*attributes, "BUSINESS_ROLE_FLAG")
```

Comment:
Are we sure there is no case where roleKc.Attributes is nil?

Reply:
Not sure why you are using the temporary variable attributes. It is the same pointer to a map, not a new map. I guess you can directly write:
roleRep.Attributes = roleKc.Attributes
delete(*roleRep.Attributes, "BUSINESS_ROLE_FLAG")
(perhaps with a test if roleRep.Attributes==nil)